src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Manage Logical Data Replication
+summary: Manage the tables that are part of the logical data replication stream.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+Once you have **logical data replication (LDR)** running, you will need to track and manage certain parts of the job:
+
+- [Conflict resolution](#conflict-resolution): As changes to a table replicate from the source to the destination cluster, there can be conflicts during some operations that the job will handle with conflict resolution. When LDR is started, the job creates a [_dead letter queue (DLQ)_](#dead-letter-queue-dlq) table with each replicating table. LDR will send any unresolved conflicts to the DLQ, which you should monitor as LDR continues to replicate changes between the source and destination clusters. 
+- [Schema changes](#schema-changes): The tables that are part of the LDR job may require schema changes, which need to be coordinated. There are some schema changes that are supported while LDR is active.
+- [Jobs](#jobs-and-ldr): Other CockroachDB jobs can operate on clusters running LDR [jobs]({% link {{ page.version.version }}/show-jobs.md %}). You may want to consider where you start and how you manage [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}), [changefeeds]({% link {{ page.version.version }}/change-data-capture-overview.md %}), [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}), and so on when you're running LDR.
+
+## Conflict resolution
+
+Conflicts in LDR are detected when there is either:
+
+- An `UPDATE` operation replicated to the destination cluster.
+- A cross-cluster write occurs, which means both clusters are writing to the same key. For example, if the LDR stream attempts to apply a row to the destination cluster where the existing row on the destination was not written by the LDR stream.
+
+LDR uses _last write wins (LWW)_ conflict resolution based on the [MVCC timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#mvcc) of the replicating write. LDR will resolve conflicts by inserting the row with the latest MVCC timestamp. When a conflict cannot apply, due to violating a foreign key constraint or schema constraint, it will be retried for up to a minute and then put in the [DLQ](#dead-letter-queue-dlq) if it could not be resolved. 
+
+### Dead letter queue (DLQ)
+
+When the LDR job starts, it will create a DLQ table with each replicating table so that unresolved conflicts can be tracked. The DLQ will contain the writes that LDR cannot apply after the retry period. For example, [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) dependendies that are not met when there are foreign key constraints in the schema.
```

Comment:
"For example, [foreign key](https://deploy-preview-19043--cockroachdb-docs.netlify.app/docs/v24.3/foreign-key) dependendies that are not met when there are foreign key constraints in the schema." 

can we add "in validated mode" because this is only a thing in validated mode?

Reply:
Added "in `validated` mode"